JUnit version 3.8.2
.E.E..E.E.E..E.E...E..E..E
Time: 9.631
There were 10 failures:
1) test08(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.UnsupportedOperationException
	at org.apache.commons.jxpath.ri.axes.RootContext.nextSet(RootContext.java:82)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:52)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test08(UnionContext_ESTest.java:205)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
2) test09(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:52)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test09(UnionContext_ESTest.java:222)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
3) test07(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
org.apache.commons.jxpath.JXPathFunctionNotFoundException: Undefined function: 
	at org.apache.commons.jxpath.ri.JXPathContextReferenceImpl.getFunction(JXPathContextReferenceImpl.java:641)
	at org.apache.commons.jxpath.ri.axes.RootContext.getFunction(RootContext.java:117)
	at org.apache.commons.jxpath.ri.compiler.ExtensionFunction.computeValue(ExtensionFunction.java:84)
	at org.apache.commons.jxpath.ri.axes.PredicateContext.nextNode(PredicateContext.java:82)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:53)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test07(UnionContext_ESTest.java:173)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
4) test00(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:52)
	at org.apache.commons.jxpath.ri.axes.NodeSetContext.nextNode(NodeSetContext.java:65)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test00(UnionContext_ESTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
5) test11(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
org.evosuite.runtime.mock.java.lang.MockIllegalArgumentException: No such variable: 'n&*0sHhje'
	at org.apache.commons.jxpath.BasicVariables.getVariable(BasicVariables.java:61)
	at org.apache.commons.jxpath.ri.model.VariablePointer.getBaseValue(VariablePointer.java:68)
	at org.apache.commons.jxpath.ri.model.VariablePointer.getLength(VariablePointer.java:132)
	at org.apache.commons.jxpath.ri.axes.InitialContext.setPosition(InitialContext.java:65)
	at org.apache.commons.jxpath.ri.axes.InitialContext.nextNode(InitialContext.java:59)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:53)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test11(UnionContext_ESTest.java:273)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
6) test12(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.ArithmeticException: / by zero
	at org.apache.commons.jxpath.ri.compiler.CoreOperationMod.computeValue(CoreOperationMod.java:37)
	at org.apache.commons.jxpath.ri.axes.PredicateContext.nextNode(PredicateContext.java:82)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:53)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test12(UnionContext_ESTest.java:296)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
7) test10(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:659)
	at java.util.ArrayList.get(ArrayList.java:435)
	at java.util.Collections$UnmodifiableList.get(Collections.java:1311)
	at org.apache.commons.jxpath.ri.axes.NodeSetContext.getCurrentNodePointer(NodeSetContext.java:48)
	at org.apache.commons.jxpath.ri.axes.SelfContext.setPosition(SelfContext.java:70)
	at org.apache.commons.jxpath.ri.axes.SelfContext.nextNode(SelfContext.java:55)
	at org.apache.commons.jxpath.ri.EvalContext.nextSet(EvalContext.java:319)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:52)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test10(UnionContext_ESTest.java:249)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
8) test05(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
org.apache.commons.jxpath.JXPathException: Undefined variable: :
	at org.apache.commons.jxpath.ri.model.VariablePointer$1.getImmediateNode(VariablePointer.java:121)
	at org.apache.commons.jxpath.ri.model.NodePointer.getNode(NodePointer.java:300)
	at org.apache.commons.jxpath.ri.model.NodePointer.getValue(NodePointer.java:220)
	at org.apache.commons.jxpath.ri.model.NodePointer.getValue(NodePointer.java:217)
	at org.apache.commons.jxpath.ri.InfoSetUtil.stringValue(InfoSetUtil.java:55)
	at org.apache.commons.jxpath.ri.InfoSetUtil.stringValue(InfoSetUtil.java:60)
	at org.apache.commons.jxpath.ri.axes.PredicateContext.nextNode(PredicateContext.java:57)
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:53)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test05(UnionContext_ESTest.java:135)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
9) test13(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.UnionContext.getDocumentOrder(UnionContext.java:42)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test13(UnionContext_ESTest.java:314)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
10) test14(org.apache.commons.jxpath.ri.axes.UnionContext_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.UnionContext.setPosition(UnionContext.java:52)
	at org.apache.commons.jxpath.ri.axes.NodeSetContext.nextNode(NodeSetContext.java:65)
	at org.apache.commons.jxpath.ri.EvalContext.constructIterator(EvalContext.java:163)
	at org.apache.commons.jxpath.ri.EvalContext.hasNext(EvalContext.java:100)
	at java.util.Iterator.forEachRemaining(Iterator.java:115)
	at org.apache.commons.jxpath.ri.axes.UnionContext_ESTest.test14(UnionContext_ESTest.java:340)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 16,  Failures: 10

